firestore/docref.go

```diff
@@ -824,3 +828,15 @@ func (it *DocumentSnapshotIterator) Next() (*DocumentSnapshot, error) {
 func (it *DocumentSnapshotIterator) Stop() {
 	it.ws.stop()
 }
+
+// WithReadOptions specifies constraints for accessing documents from the database,
+// e.g. at what time snapshot to read the documents.
+func (d *DocumentRef) WithReadOptions(opts ReadOptions) *DocumentRef {
+	d.readOption = &opts
+	return d
+}
+
+// getReadOptions gets the readOption field
+func (d *DocumentRef) getReadOptions() *ReadOptions {
```

Comment:
"Getters" in Go are always written without "get" in their name. So this would just be `readOptions()`.

https://go.dev/doc/effective_go#Getters

Reply:
Removed.